### PR TITLE
Design system button

### PR DESCRIPTION
**Pull request by ljmccode**
Button component created with the following props: variant, icon, iconSize, and onClick

Button variants include: 
- solid blue
- solid white
- solid red
- outline blue
- fw outline blue
- fw outline white
- text-only red

Screenshot 2023-05-24 at 3 05 59 PM

### CODE REVIEW

File: client/src/lib/GlobalStyles.js [hunk 1]
```diff
@@ -165,6 +165,13 @@ const GlobalStyle = createGlobalStyle`
     font-size: 1.1rem;
     padding: 12px 14px;
```
Comment (raindogg) [inline client/src/lib/GlobalStyles.js:166]: Can you bump this up to `12px 16px`?

File: client/src/lib/GlobalStyles.js [hunk 2]
```diff
@@ -165,6 +165,13 @@ const GlobalStyle = createGlobalStyle`
     font-size: 1.1rem;
```
Comment (raindogg) [inline client/src/lib/GlobalStyles.js:165]: Let's get rid of these legacy styles - the `color`, `border`, `font-size`, `margin`. `border-radius` can be increased to 4px.

**Review by raindogg — COMMENTED** [review]
Looks great! Just a few small, borderline nitpicky comments 😅

**Comment by ljmccode** [comment]
@raindogg Pushed the requested updates. Let me know if you are looking for any other changes!

**Review by raindogg — APPROVED** [review]

**Pull request merged**